src/Goteo/Library/Forms/Model/ProjectCampaignForm.php

```diff
@@ -25,23 +25,20 @@
 
 class ProjectCampaignForm extends AbstractFormProcessor implements FormProcessorInterface {
 
-    public function getConstraints($field): array
+    public function getPhoneConstraints(): array
     {
-        $constraints = [];
-
-        if($field === 'paypal') {
-            $constraints[] = new Constraints\Email([
-                'checkMX' => true,
-                'checkHost' => true
-            ]);
-        }
```

Comment:
Since this field is actually getting removed in another PR, I don't see the trouble of removing these constraint options (which are getting deprecated in this Symfony version).

For now, I'll simply remove them from this form, since it's mandatory to do so. In the other PR, the field will simply be removed from the form.

Reply:
I think we talked about this previously, but I don't remember if we reached a consensus on wether we should use the stric mode or not. Let me add that to use the strict mode we have to add a `composer require egulias/email-validator`

`
$constraints[] = new Constraints\Email([
'mode' => 'strict'
});
`

If we don't want to go with this library, I think it's interesting to, at least, use the html5 option.

Reply:
Alright, if we still wanna give support to the 'paypal' field to be saved, we should definitely keep using the `new Constraints\Email()`.

However, there's a PR to actually remove this field from the form, isn't it?

We could maybe accept this PR, letting the `new Constraints\Email()`, without the deprecated options (`checkMX` and `checkHost`, that require the Composer dependency as you've mentioned). And then, merge the PR that removes the field from the form. That way, we don't need to keep the checkMX and checkHost options. I agree with you, that leaving the `Constaints\Email` is important, as long as we keep the field in the form (and removed it thinking that we actually didn't care about the field).

Lets discuss this later today.

Reply:
Yes, you are right. I was thinking more about other forms, not in this one particularly.

We can just let it with the `Constraints\Email` and then with #388 remove this field.

Reply:
Cool! There are no other forms with the checkMX or checkHost options. So with this change we will be good to go. I'm going to commit a change with the constraint, and then we can safely publish this to production.